src/components/OrganizationListGrid.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import { ApiOrganization } from '../pdc-api';
+import { OrganizationListGridItem } from './OrganizationListGridItem';
+
+interface OrganizationListGridProps {
+	organizations: ApiOrganization[];
+	activeOrganizationId?: string | undefined;
+}
+
+export const OrganizationListGrid = ({
+	organizations,
+	activeOrganizationId = undefined,
+}: OrganizationListGridProps) => {
+	const listGridRef = React.useRef<HTMLDivElement>(null);
+
+	useEffect(() => {
+		// We want the active organization to be visible in the sidebar
+		// when the page renders, so the user knows "where they are".
+		//
+		// If we just scrolled the active organizations into view, however,
+		// the user might not realize there are more organizations
+		// "above" the active one; thus, we scroll to the previous element.
+		// If it doesn't exist -- i.e., if the first organization is active --
+		// then we can noop since it's visible by default anyway.
+		//
+		// Note that we only run this on initial page render,
+		// not every time the active organization changes.
+		// This is because we only need to orient the user on load;
+		// after that, their clicks run the show.
```

Comment:
This comment make sense but I wanted to offer a suggestion (side note -- you'll find I often make suggestions when I see comments!  It's a bit of a philosophy that a comment is USUALLY something that indicates code that isn't self-documenting)

1) Can we pull this one-liner into a named function such as `scrollToActiveItem`

2) Regarding the comment itself, though, if feels like this is a band aid (to be clear: animating the scroll sounds great / like the right UX regardless of reason!).  What I mean is: what if someone clicked and looked away, and didn't see the animation?  What if they scrolled, left the tab open, came back the next day and forgot the context?

It seems like the real solution to the problem described in the comment would be an additional UI element that indicates there is content "up".  I'm not suggesting that this get added in this PR, but *am* suggesting that we can remove the comment and instead make an issue for it.

I also think that the content of the comment probably belongs in the commit message instead.  Commit messages are contextually locked to a moment in time, whereas a comment may well get stale over time and become tech debt.

Reply:
I will say that this is a shamelessly copied comment/functionality from the current ProposalListGrid component so it might be something that @reefdog can better respond to!

Reply:
Ha I feel even stronger that it shouldn't be included then!  Strongly suggest you DRY this up and toss it into the commit message (or just don't include it)

I do think that the named helper function would be good either way though.

Reply:
Oh, actually -- can we DRY the entire concept of a list grid somehow?

It sounds like there is some redundant code; would it make sense to have some kind of generic `ListGrid` component that provides the common functionality and somehow you can override the component that actually goes inside?

Reply:
I pretty much agree with all of the above, which pains me to say. (And yes, much of it applies to my original implementation that Harry was just re-using. Eep!)

- the scroll behavior would be more self-documenting as a named function
- the overly-large and peacocking comment belonged primarily in a commit message; feel free to remove it (except the bit that belongs as part of the jsdoc)
- abstracting out a generic `ListGrid` (now that we have a second version) is probably worthwhile

Reply:
For help Harry, ListGrid, I imagine this would take two props:

```
   items T[]
   renderItem(item: T)
```

And then OrganizationListGrid would *use* ListGrid and implement the organization-specific rendering function.

You then can have all of the generic styling / listgrid functionality in `ListGrid` which gets re-used and then the organization-specific styling and functionality in OrganizationListGrid.